src/select.rs

```diff
@@ -233,6 +237,7 @@ impl<'a, T> Select<'a, T> {
 
     fn handle_left(&mut self) -> Result<(), io::Error> {
         if self.cur_page > 0 {
+            self.cursor = 0;
```

Comment:
I suppose if what this should do is move within the filter string, but I think we can punt on that

Reply:
this resets the cursor position so it is not hidden when moving to a page which has less items than the cursor is currently set to. ideally this would only move the cursor if the cursor position is bigger than the items on a page.

Reply:
in the case of filtering = true at least